src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3193,6 +3194,18 @@ void mvnw(Register Rd, Register Rm,
   INSN(sve_mls,   0b00000100, 0, 0b011); // multiply-subtract, writing addend: Zda = Zda + -Zn*Zm
 #undef INSN
 
+// SVE floating-point compare abs (predicated)
```

Comment:
This should be handled by the "SVE Integer/Floating-Point Compare - Vectors" code.